### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by AndreySlabov**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,18 +3,30 @@
...
-* [SimpleData.cs](./CS/SimpleData.cs) (VB: [SimpleData.vb](./VB/SimpleData.vb))
-* [Window1.xaml](./CS/Window1.xaml) (VB: [Window1.xaml](./VB/Window1.xaml))
-* [Window1.xaml.cs](./CS/Window1.xaml.cs) (VB: [Window1.xaml.vb](./VB/Window1.xaml.vb))
-<!-- default file list end -->
-# How to automatically create columns for all fields in a grid's datasource
+# WPF Data Grid - Generate Columns for All Fields in a Data Source
+
+This example uses the [DataControlBase.AutoGenerateColumns](https://docs.devexpress.com/WPF/DevExpress.Xpf.Grid.DataControlBase.AutoGenerateColumns) property to populate the grid with columns from its data source.
+
+The image below demonstrates the difference between generation modes (**AddNew**, **KeepOld**, and **RemoveOld**). Manually defined columns are highlighted with light yellow, automatically generated — do not have a background.
```
Comment (albertov05) [inline Readme.md]: `Manually defined columns are highlighted with light yellow, automatically generated — do not have a background.`

What do you mean by "automatically generated"? Can you clarify what is automatically generated?

Comment (AndreySlabov) [reply Readme.md]:
```suggestion
The image below demonstrates the difference between generation modes (**AddNew**, **KeepOld**, and **RemoveOld**). Manually defined columns are highlighted with light yellow, automatically generated columns do not have a background.
```
It's `automatically generated columns`
I wanted to get rid of repetition and make the sentence shorter.

Comment (albertov05) [reply Readme.md]:
```suggestion
The image below demonstrates the difference between generation modes (**AddNew**, **KeepOld**, and **RemoveOld**). Manually defined columns are highlighted in light yellow; automatically generated columns do not have a background color.
```

Comment (albertov05) [reply Readme.md]: Thanks. I slightly edited your suggestion.

**Pull request merged**